JSQMessagesViewController/Views/JSQMessagesComposerTextView.m

```diff
@@ -240,10 +246,24 @@ - (void)forwardInvocation:(NSInvocation *)anInvocation {
 }
 
 - (BOOL)canPerformAction:(SEL)action withSender:(id)sender {
+    
     if (!self.selectedTextRange.empty) {
+        
+        if ([self isTextStyleOptionsMenuItemSelected]) {
+            [UIMenuController.sharedMenuController setMenuItems:self.customTextStyleOptions];
+        }
+        
+        BOOL isActionTextStyleOptions = action == @selector(_showTextStyleOptions:);
+        BOOL areCustomTextStyleOptionsDefined = self.customTextStyleOptions != nil;
```

Comment:
need to check if they're not empty as well
BOOL areCustomTextStyleOptionsDefined = self.customTextStyleOptions.count > 0;

Reply:
Ok, thanks for noticing